### DISCUSSION TITLE
How to bypass the HTTP with azure AD consent when a user opens AA4PP ?

### DISCUSSION THREAD

**Discussion by tocri** [microsoft/powercat-creator-kit#345] [Q&A]
When a user opens AA4PP the first time, a prompt appears and the user needs to input the url https://graph.microsoft.com

![image](https://github.com/microsoft/powercat-creator-kit/assets/10110260/8179ae21-4d67-4733-b1f4-df2b55065bfb)

So i tried to bypass the consent with this helpful page https://learn.microsoft.com/en-us/powershell/module/microsoft.powerapps.administration.powershell/set-adminpowerappapistobypassconsent?view=pa-ps-latest.

But it doesn't work for this connector.

I also tried to add my AAD security group as read only users to these flows : 

* shareconnectionwithserviceprincipals
* searchaadgroup
* searchsharepointsites and List

No changes for the user.

I hope you can help me with that. I wish to bypass the consent for the users please

**Comment by mikefactorial** [comment]
@tocri Mind posting this over on https://github.com/microsoft/coe-starter-kit/issues?
